packages/block-library/src/post-tags/index.php

```diff
@@ -8,20 +8,40 @@
 /**
  * Renders the `core/post-tags` block on the server.
  *
+ * @param array $attributes The block attributes.
+ *
  * @return string Returns the filtered post tags for the current post wrapped inside "a" tags.
  */
-function render_block_core_post_tags() {
+function render_block_core_post_tags( $attributes ) {
 	$post = gutenberg_get_post_from_context();
 	if ( ! $post ) {
 		return '';
 	}
 	$post_tags = get_the_tags();
 	if ( ! empty( $post_tags ) ) {
-		$output = '';
-		foreach ( $post_tags as $tag ) {
-			$output .= '<a href="' . get_tag_link( $tag->term_id ) . '">' . $tag->name . '</a>' . ' | ';
-		}
-		return trim( $output, ' | ' );
+		$output = array_map(
```

Comment:
Do you think it'd be feasible to use `the_tags()` for rendering and just map the attributes?

Reply:
That echos the output, and we lose the ability to add more features in the future.